### PR TITLE
Add popup displaying geojson properties on click

### LINKED ISSUE #187: GeoJSON - information for a point?
Is GeoJSON supposed to show information if click / hover over a point?

For example, in jupyterlab/jupyterlab-demo, there is [a .geojson file](https://github.com/jupyterlab/jupyterlab-demo/blob/master/data/Museums_in_DC.geojson) showing museum map locations and there is a lot of information about them, but from the mapview it is hidden. Is my GeoJSON extension failing, or is it not something that is implemented?

I notice that on GitHub, the geojson previewer allows for viewing information about the point on the map.

# GitHub preview of the file

github-geojson-preview

# My GeoJSON extension installation in action
geojson-point-info-fail

### PR DESCRIPTION
**Pull request by adriantre**
fixes #187

### CODE REVIEW

**Comment by consideRatio** [comment]
GitHub tip: If you write "fixes" or "closes" instead of solves, it will close that issue along with a merge of this PR!

**Comment by adriantre** [comment]
> GitHub tip: If you write "fixes" or "closes" instead of solves, it will close that issue along with a merge of this PR!

Thanks for the tip! Will edit.

File: packages/geojson-extension/src/index.ts [hunk 1]
```diff
@@ -109,7 +109,18 @@ export class RenderedGeoJSON extends Widget implements IRenderMime.IRenderer {
         )
         .addTo(this._map);
       // Create GeoJSON layer from data and add to map
-      this._geoJSONLayer = leaflet.geoJSON(data).addTo(this._map);
+      this._geoJSONLayer= leaflet.geoJSON(data, {
+        onEachFeature: function (feature, layer) {
+          if (feature.properties) {
+            var popupContent = '<table>';
+            for (var p in feature.properties) {
+              popupContent += '<tr><td>' + p + ':</td><td><b>' + feature.properties[p] + '</b></td></tr>';
```
Comment (blink1073) [inline packages/geojson-extension/src/index.ts]: Given that this is writing to HTML, we have to sanitize the value.   See https://github.com/jupyterlab/jupyterlab/blob/72d9e17dadbd8c9e9869c863949e44b57a80120d/tests/test-apputils/src/sanitizer.spec.ts#L17 for an example.

Comment (adriantre) [reply packages/geojson-extension/src/index.ts]: Good call. I added the dependency and used the defaultSanitizer as suggested

**Comment by adriantre** [comment]
I must admit that I don't know what to do with the failing CI. Any help would be much appreciated!

**Comment by blink1073** [comment]
The failing CI is because we are in a transition state with Jupyterlab itself.  I am going to release 2.0 final of JupyterLab today, update and merge #216 , and then merge this PR once it is green.

**Pull request merged**